src/test/java/org/folio/logintest/RestVerticleTest.java

```diff
@@ -429,7 +429,7 @@ public void handle(HttpClientResponse hcr) {
     } else {
       url = okapiUrl + "/users?query=id==\"" + userId + "\"";
     }
-    return doRequest(vertx, url, HttpMethod.GET, null, null, 404,
+    return doRequest(vertx, url, HttpMethod.GET, null, null, 200,
```

Comment:
Now the mock returns 200 for a non-existing user.
Simply use testMockUser instead of failMockUser to test this.
Or change the mock to return 404 again.